administrator/components/com_associations/models/association.php

```diff
@@ -16,5 +16,27 @@
  */
 class AssociationsModelAssociation extends JModelList
 {
-
+	/**
+	 * Method to get the record form.
+	 *
+	 * @param   array    $data      Data for the form.
+	 * @param   boolean  $loadData  True if the form is to load its own data (default case), false if not.
+	 *
+	 * @return  mixed  A JForm object on success, false on failure
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getForm($data = array(), $loadData = true)
```

Comment:
$this->form->getInput('itemlanguage');
here: https://github.com/joomla-projects/gsoc16_improved-multi-lingual/pull/17/files#diff-d7a25844a430646eaa1f3071a83a58e0R209

if there's an easier way please do tell :smile: